composer/workflows/dataflowtemplateoperator_create_dataset_and_table_helper.py

```diff
@@ -0,0 +1,88 @@
+# limitations under the License.
+
+# This script is a helper function to the Dataflow Template Operator Tutorial.
+# It helps the user set up a BigQuery dataset and table that is needed
+# for the tutorial.
+
+import argparse
+
+# [START composer_dataflow_dataset_table_creation]
+
+# Make sure to follow the quickstart setup instructions beforehand.
+# See instructions here:
+# https://cloud.google.com/bigquery/docs/quickstarts/quickstart-client-libraries
+
+# Before running the sample, be sure to install the bigquery library
+# in your local environment by running pip install google.cloud.bigquery
+
+# If you choose to download the library, you can also run the sample with the
+#  following command:
+#    python dataflowtemplateoperator_create_dataset_and_table_helper.py \
+#    PROJECT_ID \
+#    LOCATION
+#   where `PROJECT_ID` is your Google Cloud Project ID and `LOCATION` is where you
+#   want the location of your BigQuery dataset.
+
+from google.cloud import bigquery
+
+
+def create_dataset_and_table(project, location, dataset_name):
+    # Construct a BigQuery client object.
+    client = bigquery.Client(project)
+
+    dataset_id = f"{project}.{dataset_name}"
+
+    # Construct a full Dataset object to send to the API.
+    dataset = bigquery.Dataset(dataset_id)
+
+    # Send the dataset to the API for creation.
+    # Raises google.api_core.exceptions.Conflict if the Dataset already
+    # exists within the project.
+    dataset = client.create_dataset(dataset)  # Make an API request.
+
+    print(f"Created dataset {client.project}.{dataset.dataset_id}")
+
+    # Create a table from this dataset.
+
+    table_id = f"{client.project}.{dataset_name}.average_weather"
+
+    schema = [
+        bigquery.SchemaField("location", "GEOGRAPHY", mode="REQUIRED"),
+        bigquery.SchemaField("average_temperature", "INTEGER", mode="REQUIRED"),
+        bigquery.SchemaField("month", "STRING", mode="REQUIRED"),
+        bigquery.SchemaField("inches_of_rain", "NUMERIC", mode="NULLABLE"),
+        bigquery.SchemaField("is_current", "BOOLEAN", mode="NULLABLE"),
+        bigquery.SchemaField("latest_measurement", "DATE", mode="NULLABLE"),
+    ]
+
+    table = bigquery.Table(table_id, schema=schema)
+    table = client.create_table(table)  # Make an API request.
+    print(f"Created table {table.project}.{table.dataset_id}.{table.table_id}")
+
+    # [END composer_dataflow_dataset_table_creation]
+    return dataset, table
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Create BQ dataset and table")
+    parser.add_argument("project", metavar="P", type=str, help="your GCP project ID")
+    parser.add_argument(
+        "location",
+        metavar="L",
+        type=str,
+        help='where your dataset should reside (i.e., "U.S."',
+    )
+    args = parser.parse_args()
+    create_dataset_and_table(args.project, args.location, "average_weather")
```

Comment:
@kurtisvg @leahecole @busunkim96 

We (Python repo owners) talked about having main around May 8th, and we decided that we don't want advocate having main in our samples.

Personally I'm ok to merge this because 1) the review happened before our decision 2) we still don't have an official recommended alternative 3) the main in this sample is simple enough to maintain or replace later

WDYT?